setup.py

```diff
@@ -622,8 +624,27 @@ def pkgconfig_win(flag, cut):
     else:
         print('no pcl install or pkg-config missed.')
         sys.exit(1)
+		
+		
+
+
+		
+if platform.system() == "Windows":	
```

Comment:
Please write it in the if - else you defined earlier.
Please Add Windows logic to the end of if.(line 505)
Please Add MacOSX/Linux logic to the end of else.(line 627)